app/assets/javascripts/client/inventories/productEntryModalCtrl.js

```diff
@@ -7,17 +7,18 @@
   ProductEntryModalCtrl.$inject = [
     '$scope',
     'ProductEntry',
-    'Enums'
+    'ConstantsService',
+    'CheckboxService'
   ];
 
-  function ProductEntryModalCtrl($scope, ProductEntry, Enums) {
+  function ProductEntryModalCtrl($scope, ProductEntry, ConstantsService, CheckboxService) {
     var vm = this;
 
     vm.closeModal = function() {
       $scope.$emit('close-product-entry-modal');
     };
 
-    vm.enums = Enums;
+    vm.constants = ConstantsService.constants;
```

Comment:
This doesn't automatically invoke the `get()` for pulling in the constants, which causes the checkboxes to not show.  This could be done in the `link` function in the directive for this, so we're not having to mess about with `$timeout`.